src/sudoers/ast.rs

```diff
@@ -48,15 +48,6 @@ impl Tag {
     }
 }
 
-impl Default for Tag {
```

Comment:
nice catch!

Reply:
it was clippy's catch. I don't think CI runs clippy in a way that makes the build fail because this Default impl became removable in PR720 but I forgot run clippy locally before opening that PR

Reply:
Yeah, clippy in the CI is a bit lightweight, sadly.